addons/core/translations/resources/en-us.yaml

```diff
@@ -833,6 +833,11 @@ credential-store:
   titles:
     new: New Credential Store
     worker-filter: Worker Filter
+    edit-worker-filter:
+      title: Edit Worker Filter
+      description: Specify workers that have access to the credential store
+    formatted-worker-filter:
+      description: The worker filter format for this credential store
```

Comment:
currently all these new translations are nested under `credential-store.titles` however, we should create a new section under `credential-store` named `worker-filter` (similar to how `brokered-credential-source` is nested under `target`). That way we can also move the `credential-store.messages.no-filter` section into `worker-filter`. In addition, the `Edit Worker Filter` translation can be moved to `credential-store.actions`.

Reply:
Thanks. Make sense!

Reply:
💡 I refactored the file with one caveat, the `Edit Worker Filter` translation should prob be moved to `credential-store.actions` for when we finally have the option to edit in the dropdown. For this feature, there is no action of editing really happening yet. Lmk what you think about these changes.